templates/instruc_template.forTasks.rst

```diff
@@ -0,0 +1,236 @@
+:orphan: true
+	 
+.. Based on: https://dmtn-030.lsst.io/v/DM-7096/index.html#task-topic-type, with learnings from the 4 sfp pages built in branch DM-8717
+
+########################
+Guidance for ExampleTask
```

Comment:
Ok (we had moved to `ExampleTask` earlier for this header, and initially this document was meant to be arrived at *from* the `ExampleTask` Template itself, but if this is the first page they are coming to, as i can see you would intend for them, then i see what you mean about how it's less descriptive your suggestion)